assets/css/pc/pages/sunglasses.css

```diff
@@ -245,6 +245,71 @@ div.color {
 .orderButton.old:hover {
 	background-color: #a6000c;
 }
+#infograph {
+    width: 100%;
+    height: 100px;
+    position: absolute;
+    z-index: 999;
+    bottom: 20px;
+    text-align: center;
+}
+#infograph .center-infograph {
+    position: relative;
+    width: auto;
+    height: auto;
+}
+#infograph .center-infograph div {
+    width: 100px;
+    height: 100px;
+    position: relative;
+    margin: 0;
+    padding: 0;
+    display: inline-block;
+    -webkit-transition: all 200ms ease-in;
+    -webkit-transform: scale(1);
+    -ms-transition: all 200ms ease-in;
+    -ms-transform: scale(1);
+    -moz-transition: all 200ms ease-in;
+    -moz-transform: scale(1);
+    transition: all 200ms ease-in;
+    transform: scale(1);
+}
+#infograph .center-infograph div:hover {
+    z-index: 2;
+    -webkit-transition: all 200ms ease-in;
+    -webkit-transform: scale(1.5);
+    -ms-transition: all 200ms ease-in;
+    -ms-transform: scale(1.5);
+    -moz-transition: all 200ms ease-in;
+    -moz-transform: scale(1.5);
+    transition: all 200ms ease-in;
+    transform: scale(1.5);
+}
+#img-1 {
+    background: url("../../../img/svg/5_years.svg");
```

Comment:
Please move images to something like `img/shop-benefits-infographic/`.
I don't care if it's `svg` or not. `img/svg/5_years.svg` is not very descriptive of what is this image for.